### PR TITLE
WIP NotReady to merge - add shirtty board - hackaday supercon 2019 addon

### PR DESCRIPTION
**Pull request by sarfata**
This is the circuitpython board for the trinket-m0 based shirtty board - as seen at Hackaday supercon 2019.

Repo for the hardware and software: https://github.com/sarfata/shirtty-addon.

@tannewt opening this as promised. Let me know if you have a doc/process on adding new boards.

### CODE REVIEW

File: ports/atmel-samd/boards/shirtty/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,15 @@
+LD_FILE = boards/samd21x18-bootloader.ld
+USB_VID = 0x239A
+USB_PID = 0x801F
```
Comment (ladyada) [inline ports/atmel-samd/boards/shirtty/mpconfigboard.mk]: USB VID and PID are something you cannot borrow from another board. So don't copy and paste these values! If you don't have a VID/PID pair, open up an issue in CircuitPython repository and request one, we will assist in the process of assigning a PID for use with CircuitPython.

https://learn.adafruit.com/how-to-add-a-new-board-to-circuitpython/customizing-the-board-files#mpconfigboard-dot-mk-5-25

Comment (sarfata) [reply ports/atmel-samd/boards/shirtty/mpconfigboard.mk]: Oops - Yes I am aware and fully understand. I wanted to get this PR opened ASAP after the conference. I will RTFM the guide on how to add a new board and update this. Changing the title to make it clear it's not ready to merge.

Comment (ladyada) [reply ports/atmel-samd/boards/shirtty/mpconfigboard.mk]: sounds good - thanks :)

**Comment by sarfata** [comment]
Closing this. I will open a new PR against master once we have the VID/PID (#2322).

**Pull request closed without merging**